.github/workflows/rust-docs-url.yml

```diff
@@ -0,0 +1,60 @@
+---
+name: Update Technical Docs Link
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - docs-v*
+  workflow_call:
+    inputs:
+      branch:
+        type: string
+        description: The branch to update the technical docs link.
+        required: true
+permissions:
+  actions: read
+  contents: write
+  pull-requests: write
+  packages: write
+  id-token: write
+jobs:
+  update-docs-link:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/create-github-app-token@3ff1caaa28b64c9cc276ce0a02e2ff584f3900c5  # v2.0.2
+        id: gh-app-token
+        with:
+          app-id: ${{ vars.GH_APP_ID }}
+          private-key: ${{ secrets.GH_APP_PRIVATE_KEY }}
+      - name: Checkout repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683  # v4.2.2
+        with:
+          token: ${{ steps.gh-app-token.outputs.token }}
+      - name: Use branch input
+        id: get_branch
+        run: |
+          echo "branch=${{ inputs.branch }}" >> $GITHUB_OUTPUT
+      - name: Update the technical docs link in nav.adoc
```

Comment:
Is it possible to use some kind of redirection instead of updating nav.adoc file every time?

Reply:
We will have different deployment url with every deploy?

Reply:
Can we do it without modifying file?

Maybe to always point to single url and in the background to redirect?

Reply:
For now, just going to replace the file. Next iteration we can add cloudflare and make redirection in the backend from github actions.